src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -30,9 +30,33 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   </Choose>
 
   <PropertyGroup>
-    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <EnableDefaultCompileItems Condition=" '$(EnableDefaultCompileItems)' == '' ">false</EnableDefaultCompileItems>                                <!--- Do not glob F# source files -->
+    <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">false</EnableDefaultNoneItems>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <DefaultProjectTypeGuid Condition=" '$(DefaultProjectTypeGuid)' == '' ">{F2A71F9B-5D33-465A-A702-920D77279786}</DefaultProjectTypeGuid>         <!-- F# project type -->
+    <PlatformTarget>AnyCPU</PlatformTarget>
```

Comment:
A bunch of these are defined in the SDK - PlatformTarget for instance is inferred taking into account RIDs and such. It\s better to not override them here. Same for DocumentationFile, DebugType, DefineConstants etc.

The current F# templates set Prefer32Bit=true. It also make Tailcalls false for Debug and true only release.